problems/loops.js

```diff
@@ -143,7 +244,24 @@ function fizzBuzzPart1() {}
  * @returns {Array}
 
  */
-function fizzBuzzPart2() {}
+    // function fizzBuzzPart2() {}
+const fizzBuzzPart2 = num => {
+  let allSumArray2 = []
+  let i = 1
+  while (i <= 100) {
+    if(i % 3 === 0 && i % 5 === 0) {
+      allSumArray2.push("FizzBuzz")
+    } else if(i % 5 === 0 && i % 3 !== 0) {
```

Comment:
No need to check the right side of &&, if it was divisible by 3, it would have satisfied the if.